app/views/users/new.html.erb

```diff
@@ -1,2 +1,19 @@
-<h1><%= t "user" %></h1>
-<p>Find me in app/views/users/new.html.erb</p>
+<% provide :title, I18n.t("signup") %>
+
+<h1><%= I18n.t("signup") %></h1>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for @user, url: signup_path do |f| %>
+      <%= render "shared/error_messages" %>
+      <%= f.label :name %>
+      <%= f.text_field :name, class: "form-control" %>
+      <%= f.label :email %>
+      <%= f.email_field :email, class: "form-control" %>
+      <%= f.label :password %>
+      <%= f.password_field :password, class: "form-control" %>
+      <%= f.label :password_confirmation, I18n.t("confirmation") %>
```

Comment:
In this section, you can study about more how you can use I18n in views
[http://guides.rubyonrails.org/i18n.html#abstracting-localized-code](http://guides.rubyonrails.org/i18n.html#abstracting-localized-code)